onnx/test/helper_test.py

```diff
@@ -271,6 +271,17 @@ def test_model_metadata_props(self):  # type: () -> None
         dupe.value = 'Other'
         self.assertRaises(checker.ValidationError, checker.check_model, model_def)
 
+    def test_model_irversion(self):  # type: () -> None
+        def test(opset_version, ir_version):  # type: (int, int) -> None
+            graph = helper.make_graph([], "my graph", [], [])
+            model = helper.make_model_gen_version(graph, opset_imports=[helper.make_opsetid("", opset_version)])
+            self.assertEqual(model.ir_version, ir_version)
+        # opset version 9 requires minimum ir_version 4, etc.
+        test(9, 4)
```

Comment:
can you add some tests with multiple imports... 
another questions : Is this acceptable ai.onnx 11, ai.onnx.ml 2. ai.onnx.training 1 ? Can you add such a test case with expected result

Reply:
I added more tests. About the second question: this will simply choose the "maximum" among the "minimum IR version" required for each opset in the list. So, it is acceptable. The rationale for this is that we should keep the IR version backward compatible as much as possible. Any model that is valid in IR version N should be a valid model in IR version n+1.